pom.xml

```diff
@@ -1718,7 +1718,6 @@
                                     <!-- -Xep:ReferenceEquality:OFF fails in IDEA --> \
                                     <!-- -Xep:BoxedPrimitiveEquality:OFF fails in IDEA --> \
                                     -Xep:GuardedBy:OFF <!-- needs some careful inspection --> \
-                                    -Xep:HashtableContains:OFF <!-- https://github.com/trinodb/trino/pull/7001 --> \
```

Comment:
Is this related to this PR?  Just checking

Reply:
It is too small to be extracted as a different PR.

Reply:
Yes, this PR fixes the issue flagged by this Error Prone check.